packages/ui/src/ui/store/actions/components/node/unrecognized-options.ts

```diff
@@ -0,0 +1,38 @@
+import {ThunkAction} from 'redux-thunk';
+
+import {RootState} from '../../../reducers';
+import {NodeUnrecognizedOptionsAction} from '../../../reducers/components/node/unrecognized-options';
+import {YTApiId, ytApiV3Id} from '../../../../rum/rum-wrap-api';
+import {NODE_UNRECOGNIEZED_OPTIONS} from '../../../../constants/components/nodes/node';
+import CancelHelper, {isCancelled} from '../../../../utils/cancel-helper';
+
+type NodeUnrcognizedOptionsThunkAction = ThunkAction<
+    any,
+    RootState,
+    unknown,
+    NodeUnrecognizedOptionsAction
+>;
+
+const cancelHeler = new CancelHelper();
+
+export function loadNodeUnrecognizedOptions(host: string): NodeUnrcognizedOptionsThunkAction {
```

Comment:
```suggestion
const cancelHeler = new CancelHelper();

export function loadNodeUnrecognizedOptions(
    host: string,
): ThunkAction<any, RootState, unknown, NodeUnrecognizedOptionsAction> {
```

Do we really need an extra type here?